src/zope/pagetemplate/pagetemplate.py

```diff
@@ -209,9 +210,8 @@ def bs(s):
                            (self._error_start, "%s: %s" %
                             sys.exc_info()[:2])) + self._text)
 
-        return bs('%s\n %s\n-->\n' % (self._error_start,
-                                      '\n'.join(self._v_errors))) + \
-            self._text
+        return bs('{}\n {}\n-->\n'.format(
+            self._error_start, '\n'.join(self._v_errors))) + self._text
```

Comment:
```suggestion
        return bs('{}\n {}\n-->\n{}'.format(
            self._error_start, '\n'.join(self._v_errors), self._text))
```

Would that be the same, right?

Reply:
Right.